### ISSUE TITLE
WalkthroughElement desperately needs "isActive" prop

### ISSUE THREAD

**Issue by Kreozot** [jasongaare/react-native-walkthrough#17]
E.g. you have WalkthroughElement in a list, so you want to show Tooltip only on the first element. But this is not the only case of course.

**Comment by jasongaare** [comment]
Would something like this work?

```
<List
  renderItem={(data, index) =>{
    if (index === 0) 
      <Tooltip><Element></Tooltip>
    else 
      <Element>
  }}
/>
```

**Comment by Kreozot** [comment]
Yes, but it is a matter of usability. There is a lot of cases like this, and sometimes it will look ugly (e.g. with a lot of props in the element).

**Comment by Aryk** [comment]
@Kreozot we just released a Walkthrough library that handles this exact use case (see the demo in the Readme):

https://github.com/tribefyhq/react-native-interactive-walkthrough
